### ISSUE TITLE
research

### ISSUE THREAD

**Issue by Sebastianrix** [Sebastianrix/CNN_U-Net_GAN_Project#1]
Validation Precision-Recall / Training vs Validation Loss / PR-AUC 11)

 predictive parity / Statistical parity  14)

Confusion matrix 14), 11)

10) recommender system

Models : KNN / XGBoost / XGBRegressor / Linear Regression / AdaBoost / RandomForestRegressor / Ridge Regressor

 / Rolling (mean over a perioede)(lag, the value x ago) 6), 9)

**Comment by Sebastianrix** [comment]
statistics 3) Shapiro-Wilk / Mann whitney-U / t-test / ANOVA - Something with distribution of data.
